modules/tinymce/src/core/demo/html/shadow_dom_demo.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>TinyMCE Shadow DOM Demo Page</title>
```

Comment:
Maybe add the following, so you can test mobile on here as well?
```
<meta name="viewport" content="width=device-width, initial-scale=1">
```

Reply:
Done.